worlds/sm64ex/__init__.py

```diff
@@ -80,7 +80,7 @@ def generate_basic(self):
                         self.world.MIPS1Cost[self.player].value, self.world.MIPS2Cost[self.player].value,
                         self.world.StarsToFinish[self.player].value)
         self.world.itempool += [staritem for i in range(0,starcount)]
-        mushroomitem = self.create_item("1Up Mushroom") 
+        mushroomitem = self.create_item("1Up Mushroom")
         self.world.itempool += [mushroomitem for i in range(starcount,120 - (15 if not self.world.EnableCoinStars[self.player].value else 0))]
```

Comment:
```suggestion
        self.world.itempool += [self.create_item("1Up Mushroom") for i in range(starcount,120 - (15 if not self.world.EnableCoinStars[self.player].value else 0))]
```

Only noticed this now, but you do actually need unique Mushroom Item objects, or their .location reference will be shared and very wrong.